### PR TITLE
[NTR-139] Add cookies page

### PR DESCRIPTION
**Pull request by likeuwill**
## Motivation and Context

[JIRA](https://eaflood.atlassian.net/browse/NTR-139)
## Screenshots (if appropriate):
cookies

## Checklist:

- [x] It contains only changes required by issue (does not contain other PR)
- [x] Includes link to a issue (if apply)
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: app/views/layouts/_cookies_banner.haml [hunk 1]
```diff
@@ -2,7 +2,7 @@
   .govuk-width-container
     %p
       GOV.UK uses cookies to make the site simpler.
-      %a.govuk-link{ href: 'https://www.gov.uk/help/cookies', target: '_blank' }
+      %a.govuk-link{ href: cookies_path, target: '_blank' }
```
Comment (miqs1992) [inline app/views/layouts/_cookies_banner.haml]: remove target _blank -> now it links inside out page

File: app/views/cookies/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          %p the pages you visit on GOV.UK and how long you spend on them
+        %li
+          %p what you click on while you’re visiting the site
+      %p Google Analytics sets the following cookies.
+
+      %table.govuk-table
+        %caption.govuk-table__caption
+        %thead.govuk-table__head
+          %tr.govuk-table__row
+            %th.govuk-table__header{:scope => "col"} Name
```
Comment (miqs1992) [inline app/views/cookies/index.html.haml]: use `scope: 'col'`

File: app/views/cookies/index.html.haml [hunk 3]
```diff
@@ -0,0 +1,70 @@
+%main.govuk-main-wrapper#main-content{role: 'main'}
+  .govuk-grid-row
+    .govuk-grid-column-full
+      %h1.govuk-heading-l Details about cookies on GOV.UK
```
Comment (miqs1992) [inline app/views/cookies/index.html.haml]: on Centralised Taxi and PHV Data Maintenance not GOV.UK

File: features/cookies.feature [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Feature: Cookies
+  In order to read the page
+  As a Licensing Authority
+  I want to see cookies page
+
+  Scenario: User see cookies page
+    Given I am on the Sign in page
+    When I press "Cookies" link
+    Then I should see "Details about cookies on GOV.UK"
```
Comment (miqs1992) [inline features/cookies.feature]: Needs to be adjusted to Centralised Taxi and PHV Data Maintenance

File: features/cookies.feature [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Feature: Cookies
+  In order to read the page
+  As a Licensing Authority
+  I want to see cookies page
+
+  Scenario: User see cookies page
+    Given I am on the Sign in page
+    When I press "Cookies" link
```
Comment (miqs1992) [inline features/cookies.feature]: then I press "Find out more"

**Comment by likeuwill** [comment]
@miqs1992 done

**Comment by miqs1992** [comment]
https://github.com/miqs1992/CSV-Uploader/pull/14/

**Pull request closed without merging**